src/pages/AppDetail.svelte

```diff
@@ -43,6 +43,7 @@
   <TabGroup active="Metrics">
     <Tab key="Metrics">Metrics</Tab>
     <Tab key="Pings">Pings</Tab>
+    <Tab key="Application IDs">Application IDs</Tab>
```

Comment:
I'm wondering if we should add more context to what "application IDs" means here, maybe with a tooltip? I remember having to ask you what the application IDs refer to in our last 1:1, and I'm assuming there could be other users who are also not as familiar with this.

Reply:
That's right, yeah, I was thinking of adding help tooltips for all three of these things (metrics, pings, and application ids). I was thinking maybe it would be best to do that in a followup?